packages/react/src/components/DataTable/__tests__/__snapshots__/DataTable-test.js.snap

```diff
@@ -483,7 +483,6 @@ exports[`DataTable behaves as expected selection should render and match snapsho
           role="search"
         >
           <div
-            aria-labelledby="custom-id"
```

Comment:
could we actually pass the `onExpand` prop so that this keeps the aria-labelledby instead? TIA!

Reply:
Hello @francinelucca 

Are you asking to refactor the `TableToolbarSearch` component, removing all of the `onExpand` handling logic that exists there and simply pass it through to the `Search` sub-component?

This change doesn't feel trivial to me. We will have to ensure that there are no deleterious effects.

Reply:
Hi @sjbeatle sorry I'm reading this back and realizing my explanation wasn't very clear.
I meant in the tests where `aria-labelledby` is being removed from the snapshots, if we send in an `onExpand={() => {}}` empty prop function to the `<ToolbarSearch />` in the test file, then the `aria-labelledby` won't be removed from the snapshot and we can continue to ensure that it is there when the onExpand prop is passed in.

Reply:
ah I didn't realize it was being filtered out by the `TableToolbarSearch`, we can leave it as is then, thanks for explaining that!